kbase-extension/static/kbase/js/kbaseNarrative.js

```diff
@@ -187,36 +186,54 @@ define([
     };
 
     Narrative.prototype.getUserPermissions = function () {
-        return new Workspace(Config.url('workspace'), {token: this.getAuthToken()})
-            .get_workspace_info({id: this.workspaceId})
+        return new Workspace(Config.url('workspace'), {
+            token: this.getAuthToken(),
+        })
+            .get_workspace_info({ id: this.workspaceId })
```

Comment:
the indent here is a little confusing! presumably that's `prettier` at work?

Reply:
I agree! That's good ol' eslint trying its best. :) I'll rework to make, er, prettier.